pkg/mountutil/mountutil_freebsd.go

```diff
@@ -66,3 +66,8 @@ func ProcessFlagTmpfs(s string) (*Processed, error) {
 func ProcessFlagMount(s string, volStore volumestore.VolumeStore) (*Processed, error) {
 	return nil, errdefs.ErrNotImplemented
 }
+
+func ProcessSplit(s string, volStore volumestore.VolumeStore, res Processed, src string, dst string, options []string) (string, string, []string, error) {
+	var x []string
+	return "", "", x, nil
+}
```

Comment:
freebsd implementation is the same as linux, why do we have it duplicated? (see other comment about the goos check for processsplit in mountutil.go)